third_party/3/jwt/algorithms.pyi

```diff
@@ -0,0 +1,3 @@
+from typing import Any
+
+class Algorithm: Any  # type: ignore
```

Comment:
This just makes the class body be `Any`. You need to write `class Algorithm(Any): ...` to inherit from Any.